components/post_view/post_info/post_info.tsx

```diff
@@ -320,17 +326,21 @@ export default class PostInfo extends React.PureComponent<Props, State> {
         }
     }
 
-    componentDidUpdate(prevProps: Props) {
-        const {shortcutReactToLastPostEmittedFrom, isLastPost} = this.props;
+    componentDidUpdate(prevProps: Props): void {
+        const {shortcutReactToLastPostEmittedFrom, isLastPost, isPostBeingEdited} = this.props;
 
         const shortcutReactToLastPostEmittedFromCenter = prevProps.shortcutReactToLastPostEmittedFrom !== shortcutReactToLastPostEmittedFrom &&
         shortcutReactToLastPostEmittedFrom === Locations.CENTER;
         if (shortcutReactToLastPostEmittedFromCenter && isLastPost !== undefined) {
             this.handleShortcutReactToLastPost(isLastPost);
         }
+
+        if (!prevProps.isPostBeingEdited && isPostBeingEdited) {
+            this.props.handleDropdownOpened(false);
```

Comment:
realtively simple answer ... we need to update the dropdownState to being closed ... otherwise it would stay open after we finish the post editing ... we could rename that to be more descriptive, but IMO this adds another layer of abstraction that is not really needed